internal/pkg/cli/mocks/mock_interfaces.go

```diff
@@ -3176,10 +3176,10 @@ func (m *MockpipelineStatusDescriber) EXPECT() *MockpipelineStatusDescriberMockR
 }
 
 // Describe mocks base method
-func (m *MockpipelineStatusDescriber) Describe() (*describe.PipelineStatusDesc, error) {
+func (m *MockpipelineStatusDescriber) Describe() (*describe.PipelineStatus, error) {
```

Comment:
You'll need to regenerate the mocks to get rid of this now that you've removed the interfaces from `cli/interfaces.go`

Reply:
Ah, yes. Thanks!